### PR TITLE
KOGITO-5564 Use Quarkus datasource config for Postgresql example

### PR DESCRIPTION
**Pull request by cristianonicolai**
Assembly:
- https://github.com/kiegroup/kogito-examples/pull/815
- https://github.com/kiegroup/kogito-runtimes/pull/1472
- https://github.com/kiegroup/kie-docs/pull/3812

### CODE REVIEW

**Comment by cristianonicolai** [comment]
@sutaakar @spolti @radtriste this change means that we wont use the `kogito.persistence.postgresql.connection.uri` property for Quarkus. Springboot still works the same way.
So Quarkus datasource properties now apply https://quarkus.io/guides/reactive-sql-clients#reactive-datasource
Im guessing this can have some side effects on images and operator

**Comment by radtriste** [comment]
cc @vaibhavjainwiz

File: process-postgresql-persistence-quarkus/src/main/resources/postgresql.properties [hunk 1]
```diff
@@ -5,8 +5,9 @@ quarkus.native.native-image-xmx=4g
 
 kogito.persistence.type=postgresql
 
-# Details https://www.postgresql.org/docs/9.6/static/libpq-connect.html#LIBPQ-CONNSTRING
-kogito.persistence.postgresql.connection.uri=postgresql://kogito-user:kogito-pass@localhost:5432/kogito
+quarkus.datasource.username=kogito-user
+quarkus.datasource.password=kogito-pass
```
Comment (radtriste) [inline process-postgresql-persistence-quarkus/src/main/resources/postgresql.properties:9]: forcing the username/password ?

Comment (cristianonicolai) [reply process-postgresql-persistence-quarkus/src/main/resources/postgresql.properties:9]: @radtriste that is set in sync with the settings for the postgresql database that is started as part of the docker compose, so it works out-of-the-box

**Comment by vaibhavjainwiz** [comment]
> @sutaakar @spolti @radtriste this change means that we wont use the `kogito.persistence.postgresql.connection.uri` property for Quarkus. Springboot still works the same way.
> So Quarkus datasource properties now apply https://quarkus.io/guides/reactive-sql-clients#reactive-datasource
> Im guessing this can have some side effects on images and operator

@cristianonicolai 
As I know, currently postgresql configuration could be define in runtime using 2 ways :
- Eclipse Vert.x
```
# Same for both Quarkus and Spring Boot
PGUSER
PGHOST
PGPASSWORD
PGDATABASE
PGPORT
```
- Using JDBC
```
# On Quarkus
QUARKUS_DATASOURCE_DB-KIND
QUARKUS_DATASOURCE_USERNAME
QUARKUS_DATASOURCE_PASSWORD
QUARKUS_DATASOURCE_JDBC_URL
# On Spring Boot
SPRING_DATASOURCE_USERNAME
SPRING_DATASOURCE_PASSWORD
SPRING_DATASOURCE_URL
```

Does Quarkus reactive client is different from them?

**Comment by cristianonicolai** [comment]
Jenkins rerun examples tests

**Comment by cristianonicolai** [comment]
@vaibhavjainwiz Eclipse Vert.x will only apply to SpringBoot now.
For Quarkus, we need to use
QUARKUS_DATASOURCE_USERNAME
QUARKUS_DATASOURCE_PASSWORD
QUARKUS_DATASOURCE_REACTIVE_URL
and anything else that is available from https://quarkus.io/guides/reactive-sql-clients#reactive-datasource

**Comment by cristianonicolai** [comment]
JDBC remains the same

**Comment by sutaakar** [comment]
From images/operator perspective it shouldn't have any effect AFAIK, as all the configuration is done by user. Images/operator don't have any code specific to the PostgreSQL configuration.

**Review by radtriste — APPROVED** [review]

**Review by tiagodolphine — APPROVED** [review]
LGTM, but we need a follow-up PR to update the kie-docs as well, in case we don't have it already.

**Comment by radtriste** [comment]
jenkins rerun examples tests

**Comment by cristianonicolai** [comment]
Jenkins rerun examples tests

**Pull request merged**